gocourse04/main.go

```diff
@@ -0,0 +1,168 @@
+package main
+
+import "fmt"
+
+// Наприклад, зоопарк ділиться на території: копитні, пернаті, примати… та інше.
+
+const (
+	Ungulates AnimalType = "Ungulates"
+	Feathered AnimalType = "Feathered"
+	Mammals   AnimalType = "Mammals"
+	Horses    Breed      = "Horses"
+	Pony      Breed      = "Pony"
+	Ducks     Breed      = "Ducks"
+	Warthoag  Breed      = "Warthoag"
```

Comment:
Помилочка:
```suggestion
	Warthog  Breed      = "Warthog"
```

https://en.wikipedia.org/wiki/Warthog

Reply:
Ось це не виправлено, тому не резолвлю.